### ISSUE TITLE
InsufficientCapacityOnHost

### ISSUE THREAD

**Issue by yantx-github** [aws-samples/aws-snow-tools-for-eks-anywhere#17]
When I run create-eks-a-admin-instance.sh
I got the error:
An error occurred (InsufficientCapacityOnHost) when calling the RunInstances operation: Snowball has insufficient capacity to fulfill this request
I have enough vCPUs and memories on the Snow device

**Comment by cmengtin** [comment]
There is no ssd left in your device, please delete unused images, files and instances to make room.
```
  "DeviceCapacities" : [ {
    "Name" : "HDD Storage",
    "Unit" : "Byte",
    "Total" : 0,
    "Available" : 0
  }, {
    "Name" : "SSD Storage",
    "Unit" : "Byte",
    "Total" : 194027315200,
    "Available" : 0
  }, {
    "Name" : "vCPU",
    "Unit" : "Number",
    "Total" : 52,
    "Available" : 48
  }, {
    "Name" : "Memory",
    "Unit" : "Byte",
    "Total" : 375809638400,
    "Available" : 358629769216
  }, {
    "Name" : "GPU",
    "Unit" : "Number",
    "Total" : 0,
    "Available" : 0
  } ],
```

**Comment by yantx-github** [comment]
Thank you!
